Eli250/my-brand
# kALNzQlj Contact and signin page

### What does this PR do?
     Create signin and contact page
### Description of Task to be completed?

    This PR creates the contact and sigin page for the admin panel.

### How should this be manually tested?

    Check if all text inputs are provided and validated
    Entering wrong input an check if the system shows error message
    Enter credential and check if you reach the admin page


### Relevant pivotal tracker stories?

kALNzQlj
    

### Screenshots
![Screenshot from 2021-12-23 23-22-05](https://user-images.githubusercontent.com/60324751/147292511-4db11d7b-2f3d-4c23-8ab2-3d76b24eba45.png)
![Screenshot from 2021-12-23 23-22-13](https://user-images.githubusercontent.com/60324751/147292517-adccf8f2-6c1d-4e32-a84f-300be2348b76.png)
